### PR TITLE
Also migrate `csv-remote-import` old plugin names

### PR DESCRIPTION
**Pull request by evantahler**
Update the first CSV name migration to include a change from https://github.com/grouparoo/grouparoo/pull/2293 that didn't have a migration.  

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Review by pedroslopez — APPROVED** [review]
Note: if someone had upgraded to v0.7.1 before this fix and are affected, they'll need to manually patch their csv sources or delete the `000001-plugin-csv-import-name` row from SequelizeMeta and restart.

**Comment by evantahler** [comment]
> Note: if someone had upgraded to v0.7.1 before this fix and are affected, they'll need to manually patch their csv sources or delete the `000001-plugin-csv-import-name` row from SequelizeMeta and restart.

Yeah - that's why I want to get this out fast!

File: plugins/@grouparoo/csv/src/migrations/000001-plugin-csv-import-name.ts [hunk 1]
```diff
@@ -8,6 +8,11 @@ export default {
...
       { type: "csv-import" }
     );
+    await queryInterface.bulkUpdate(
+      "sources",
+      { type: "csv-import-table" },
+      { type: "csv-remote-import" }
+    );
   },
 
   down: async (queryInterface: Sequelize.QueryInterface) => {
```
Comment (pedroslopez) [inline plugins/@grouparoo/csv/src/migrations/000001-plugin-csv-import-name.ts:18]: oop, should we add a down migration?

Comment (evantahler) [reply plugins/@grouparoo/csv/src/migrations/000001-plugin-csv-import-name.ts:18]: I don't think so... 
Ideally, this migration would be 0002 and there would have been a 0001 that change the name a first time, but that was so long ago, I'm worried what a "down" would do in that case.  

This "down" leaves you where you should be at the point this migration was introduced, which was "ahead" of the missing first migration.

**Pull request merged**